### ISSUE TITLE
how to inject a config var into a js file ?

### ISSUE THREAD

**Issue by chrisvidal** [tightenco/jigsaw#50]
I'd like to inject the value of {{ $asset_prefix }} define in config.php / config.production.php into a separate config.js file for example:

'app.config = {
    server_url: '{{ $asset_prefix }}',
    default_language: 'en'
};'

is there a way to achieve this ? thanks

**Comment by mattstauffer** [comment]
Is there any reason you don't want to set `app.config` in the head of your file?

E.g.:

```html
<!DOCTYPE html>
<html>
<head>
    <!-- other head stuff... -->
    <script>
        var app = {
            server_url: '{{ $asset_prefix }}',
            default_language: 'en'
        };
    </script>
    <script src="app.js"></script>
```

That's not to say it isn't possible; I imagine Jigsaw may be able to do it with some tweaks (or it may be even possible now, although I don't think so?). Just curious about the need.

**Comment by chrisvidal** [comment]
it is possible indeed to do it like this, in the head. i tried inside the js and it does not work so far.
my need is that i m bundling a set of js files and would prefer to keep things at their logical place for easier maintenance. Does it make sense ?

**Comment by mattstauffer** [comment]
@chrisvidal Closed; captured the concept in #56. Thanks!
